ignite/base/events_driven.py

```diff
@@ -65,13 +66,15 @@ def __init__(self) -> None:
         self.logger = logging.getLogger(__name__ + "." + self.__class__.__name__)
 
     def register_events(
-        self, *event_names: Union[List[str], List[EventEnum]], event_to_attr: Optional[dict] = None
+        self,
+        *event_names: "Union[List[str], List[ignite.engine.events.EventEnum]]",
```

Comment:
Doing this as string does not look good. Let's find another solution

Reply:
Can you please explain your point?